tools/execute-selenium-tests

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -ex
+
+CURRENT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_HOME="$CURRENT_PATH/.."
+FOXBOX_BINARY="$PROJECT_HOME/target/debug/foxbox"
+
+DEBUG_PORT=${BOX_PORT:-3331}
+DEBUG_PROFILE="$HOME/.local/share/foxbox-debug"
+
+# clean up residual profile
+rm -rf ${DEBUG_PROFILE}
+
+# build and launch foxbox daemon
+pushd ${PROJECT_HOME}
+cargo build
+$FOXBOX_BINARY -p ${DEBUG_PORT} --profile ${DEBUG_PROFILE} &
+FOXBOX_PID=$!
+while ! nc -z localhost ${DEBUG_PORT}; do sleep 3; done
+popd
+
+# run tests
+BOX_PORT=${DEBUG_PORT} ${PROJECT_HOME}/node_modules/.bin/mocha ${PROJECT_HOME}/test/selenium/*_test.js
+
+# clean up foxbox process
+kill $FOXBOX_PID
```

Comment:
Strange, so when I execute the foxbox, it dies with the error 
thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Custom(Custom { kind: Other, error: StringError("failed to lookup address information: nodename nor servname provided, or not known") }) }', ../src/libcore/result.rs:746

This was fixed when I deleted the -l localhost parameter and issued following command instead:
/Users/g5njpark/GitRepo/foxbox/tools/../target/debug/foxbox -p 3331 --disable-tls --profile /Users/g5njpark/.local/share/foxbox-debug

Ah, so -l is for linux only.  I guess it's good for the travis since it's on Ubuntu, but would there be a quick way to check for the environment in ./execute-selenium-tests?  if so we can also use npm run selenium-tests on mac.